gulpfile.js

```diff
@@ -57,7 +47,14 @@ gulp.task('install-azure-account', () => {
     }
 });
 
-function spawn(command, args, options, cb) {
+gulp.task('test', gulp.series('install-azure-account', async () => {
+    env.DEBUGTELEMETRY = 1;
+    env.CODE_TESTS_WORKSPACE = path.join(__dirname, 'test/test.code-workspace');
+    env.CODE_TESTS_PATH = path.join(__dirname, 'dist/test');
+    return spawn('node', ['./node_modules/vscode/bin/test'], { stdio: 'inherit', env });
+}));
+
+function spawn(command, args, options) {
```

Comment:
Is the return type a promise? I presume that given the use of async in the webpack tasks. No a strong preference - annotating the return type might help here

Reply:
Removed the await, thanks.
Unfortunately this is JS, not TS, so no types (at least not in the friendly TS way).